docs/samples/bookinfo.md

```diff
@@ -110,54 +56,70 @@ _Note: The following instructions assume that you have access to a kubernetes cl
+kubernetes                 10.0.0.1     <none>        443/TCP              14d
+productpage                10.0.0.120   <none>        9080/TCP             6m
+ratings                    10.0.0.15    <none>        9080/TCP             6m
+reviews                    10.0.0.170   <none>        9080/TCP             6m
+```
 
    and
 
-   ```bash
-   $ kubectl get pods
-   NAME                                        READY     STATUS    RESTARTS   AGE
-   details-v1-1520924117-48z17                 2/2       Running   0          6m
-   istio-ingress-controller-3181829929-xrrk5   1/1       Running   0          8m
-   istio-manager-175173354-d6jm7               2/2       Running   0          8m
-   istio-mixer-3883863574-jt09j                2/2       Running   0          8m
-   productpage-v1-560495357-jk1lz              2/2       Running   0          6m
-   ratings-v1-734492171-rnr5l                  2/2       Running   0          6m
-   reviews-v1-874083890-f0qf0                  2/2       Running   0          6m
-   reviews-v2-1343845940-b34q5                 2/2       Running   0          6m
-   reviews-v3-1813607990-8ch52                 2/2       Running   0          6m
-   ```
+```bash
+$ kubectl get pods
+NAME                                        READY     STATUS    RESTARTS   AGE
+details-v1-1520924117-48z17                 2/2       Running   0          6m
+istio-ingress-controller-3181829929-xrrk5   1/1       Running   0          8m
+istio-manager-175173354-d6jm7               2/2       Running   0          8m
+istio-mixer-3883863574-jt09j                2/2       Running   0          8m
+productpage-v1-560495357-jk1lz              2/2       Running   0          6m
+ratings-v1-734492171-rnr5l                  2/2       Running   0          6m
+reviews-v1-874083890-f0qf0                  2/2       Running   0          6m
+reviews-v2-1343845940-b34q5                 2/2       Running   0          6m
+reviews-v3-1813607990-8ch52                 2/2       Running   0          6m
+```
 
 1. Determine the Gateway ingress URL
 
-   ```bash
-   $ export GATEWAY_URL=$(kubectl get po -l infra=istio-ingress-controller -o jsonpath={.items[0].status.hostIP}):$(kubectl get svc istio-ingress-controller -o jsonpath={.spec.ports[0].nodePort})
-   $ echo $GATEWAY_URL
-   192.168.99.100:32567
-   ```
+```bash
+export GATEWAY_URL=$(kubectl get po -l infra=istio-ingress-controller -o jsonpath={.items[0].status.hostIP}):$(kubectl get svc istio-ingress-controller -o jsonpath={.spec.ports[0].nodePort})
+```
 1. If you open the Bookinfo URL (http://$GATEWAY_URL/productpage) in your browser, you should see the bookinfo application productpage displayed.
+```bash
+curl -v http://$GATEWAY_URL/productpage
+* Hostname was NOT found in DNS cache
+*   Trying 104.197.226.87...
+* Connected to 104.197.226.87 (104.197.226.87) port 80 (#0)
+> GET /productpage HTTP/1.1
+> User-Agent: curl/7.35.0
+> Host: 104.197.226.87
+> Accept: */*
+>
+< HTTP/1.1 500 Internal Server Error
```

Comment:
HTTP 500?
Do we need to show command line output in a demo which has a webpage or even doing curl on the CLI?